index.js

```diff
@@ -0,0 +1,94 @@
+// First name, Middle name, Last name, email, phone number, role, address
+data = [
```

Comment:
Data should be in a form of json file. Make a separate json file and populate data from that.